Add plugin: Tab Selector

namikaze-40p opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin:

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [x]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
[this.addRibbonIcon('file-check-2', 'Tab Selector', () => this.openTabSelectorModal());](https://github.com/namikaze-40p/obsidian-tab-selector/blob/97527602a278801622fca59f42d3431dcc97e7c7/src/main.ts#L12)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

["minAppVersion": "0.15.0",](https://github.com/namikaze-40p/obsidian-tab-selector/blob/97527602a278801622fca59f42d3431dcc97e7c7/manifest.json#L5)
You are using API's that are newer than this, set this to the latest public release.

namikaze-40p commented:
@joethei 
Thanks for the thorough review!
I think I have made all the necessary changes, so please review again.

---
> [this.addRibbonIcon('file-check-2', 'Tab Selector', () => this.openTabSelectorModal());](https://github.com/namikaze-40p/obsidian-tab-selector/blob/97527602a278801622fca59f42d3431dcc97e7c7/src/main.ts#L12)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

Fixed in [this commit](https://github.com/namikaze-40p/obsidian-tab-selector/commit/347a012a0dd61d25cf1ba4ab856807cd9ba45373).

---
> ["minAppVersion": "0.15.0",](https://github.com/namikaze-40p/obsidian-tab-selector/blob/97527602a278801622fca59f42d3431dcc97e7c7/manifest.json#L5)
You are using API's that are newer than this, set this to the latest public release.

Fixed in [this commit](https://github.com/namikaze-40p/obsidian-tab-selector/commit/33e6a5e5fbab888b2e2b4a8e88f585e30d4e99b1).
